<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AI_CarProgram</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-30</y>
     <width>1431</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-image: url(:/backimage/Snipaste_2021-12-04_21-15-09.png);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;请面向摄像头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_video">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;               
                   border-width:10px;                    
                                 
                   border-color:rgba(255,255,255,255);    

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Face">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#pushButton_Face{
    border-radius: 20px;
    color: white;
    padding: 16px 32px;
    text-align: center;
    text-decoration: none;
    font-size: 40px;
    margin: 4px 2px;
    background-color: white;
    color: black;
    border: 2px solid rgba(103,225,255,255);
}
QPushButton#pushButton_Face:hover {
    background-color:rgba(103,225,255,255);
    color: white;
}
QPushButton#pushButton_Face:pressed {
    background-color: rgba(19,30,95,255);
}</string>
    </property>
    <property name="text">
     <string>人脸开锁：关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#pushButton_close{
    border-radius: 20px;
    color: white;
    padding: 16px 32px;
    text-align: center;
    text-decoration: none;
    font-size: 40px;
    margin: 4px 2px;
    background-color: white;
    color: black;
    border: 2px solid rgba(103,225,255,255);
}
QPushButton#pushButton_close:hover {
    background-color:rgba(103,225,255,255);
    color: white;
}
QPushButton#pushButton_close:pressed {
    background-color: rgba(19,30,95,255);
}</string>
    </property>
    <property name="text">
     <string>关门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#pushButton_exit{
    border-radius: 20px;
    color: white;
    padding: 16px 32px;
    text-align: center;
    text-decoration: none;
    font-size: 40px;
    margin: 4px 2px;
    background-color: white;
    color: black;
    border: 2px solid rgba(103,225,255,255);
}
QPushButton#pushButton_exit:hover {
    background-color:rgba(103,225,255,255);
    color: white;
}
QPushButton#pushButton_exit:pressed {
    background-color: rgba(19,30,95,255);
}</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_person">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>190</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel#label_person{
    border-radius: 20px;
    color: white;
    padding: 16px 32px;
    text-align: center;
    text-decoration: none;
    font-size: 40px;
    margin: 4px 2px;
    background-color: rgba(103,225,255,255);;
    color: black;
    border: 2px solid rgba(103,225,255,255);
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/backimage_form.qrc"/>
 </resources>
 <connections/>
</ui>
